components/proxy/src/main/java/com/hotels/styx/admin/handlers/OriginsInventoryHandler.java

```diff
@@ -63,14 +64,8 @@ public OriginsInventoryHandler(EventBus eventBus) {
         eventBus.post(new GetOriginsInventorySnapshot());
     }
 
```

Comment:
Looks like methods have been rearranged here, making the diff less readable. `doHandle` should be just below the constructor (it also satisfied our code convention).

Reply:
Perhaps this comment should have been left against some other file?